sources/packages/api/src-sql/Types/Create-restriction-notification-type.sql

```diff
@@ -0,0 +1,2 @@
+-- Create an enum for restriction notification type.
+CREATE TYPE sims.restriction_notification_type AS ENUM ('No effect', 'Warning', 'Error');
```

Comment:
The types that we creates so far have the "s" in the end, as shown below. Can we change it to be aligned with the existing ones?
![image](https://user-images.githubusercontent.com/61259237/168680317-1c46910c-0088-4584-960d-d4551da91c6c.png)

Reply:
updating both restriction_action_types and restriction_notification_types